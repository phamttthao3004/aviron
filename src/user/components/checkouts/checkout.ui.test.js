import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import CheckoutUI from './checkout.ui';

describe('Checkout UI', () => {
    it('Render Checkout UI component', () => {
        const props = {};
        const com = shallow(<CheckoutUI {...props} />);
        expect(shallowToJson(com)).toMatchSnapshot();
    });
});
