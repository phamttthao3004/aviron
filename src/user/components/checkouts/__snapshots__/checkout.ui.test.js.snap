// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout UI Render Checkout UI component 1`] = `
<div
  className="payment-container"
  id="checkout-page"
>
  <div
    className="container"
  >
    <div
      className="warning"
    >
      <div>
        <i
          className="icon-ic_warning_24px"
        />
        <span>
          Transaction expired: over 10 munites for transaction
        </span>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-sm-7 col-xs-12"
      >
        <Route
          component={[Function]}
          path="/checkout/checkout-information"
        />
        <Route
          component={[Function]}
          path="/checkout/checkout-payment-method"
        />
        <Route
          component={[Function]}
          path="/checkout/checkout-review-order"
        />
      </div>
      <div
        className="col-sm-5 col-xs-12"
      >
        <div
          className="aviron-cart-detail"
        >
          <div
            className="title"
          >
            SUMMARY
          </div>
          <CartBox />
        </div>
      </div>
    </div>
  </div>
</div>
`;
